YuliiaShaposhnikova/HW11/Task1_calc.py

```diff
@@ -0,0 +1,12 @@
+def what_number():
```

Comment:
good job
but look at this implementation

def check_number(number):
    if number < 0:
        raise ValueError("That is not a positive number!")

    if number % 2 == 0:
        print(f"Your age {number} is an even number")
    else:
        print(f"Your age {number} is an odd number")


def what_number():
    try:
        number = int(input("Enter your age: "))
        check_number(number)
    except ValueError as e:
        print(e)